SeaBattle/SeaBattle.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Ok, we got the project, where is the solution? (.sln)

Reply:
I will push solution in next PR is it okay? Forgot about it because tryied to fix Git and pushed only git add *classes and csproj*.

Reply:
why not in this?

Reply:
You have to resolve this questions in this PR, while it's not merged yet.